boat_simulator/nodes/physics_engine/force_computation.py

```diff
@@ -0,0 +1,43 @@
+"""Class to compute lift and drag forces acting on a medium."""
+
+from typing import Tuple
+
+import numpy as np
+from numpy.typing import NDArray
+
+from boat_simulator.common.types import Scalar
+
+
+class ForceComputation:
+    def __init__(self, lift_coefficient: Scalar, area: Scalar, fluid_density: Scalar):
+        self.__lift_coefficient = lift_coefficient
+        self.__area = area
+        self.__fluid_density = fluid_density
+
+    def compute(self, apparent_velocity: NDArray) -> Tuple[NDArray, NDArray]:
+        """Computes the lift and drag forces acting on a medium.
+
+        Args:
+            apparent_velocity (NDArray): Apparent (relative) velocity between the fluid and the
+                medium, given in meters per second (m/s).
```

Comment:
Might be worth mentioning three more things:

1. `apparent_velocity = fluid_velocity - medium_velocity`
2. This class assumes a specific orientation of the medium when we compute lift and drag. This means that the apparent velocity vector needs to be rotated ahead of time to preserve the same angle of attack.
3. The medium reference vector should be defined. Maybe [1, 0]